### PR TITLE
[Fix] README.md and remove references to preview 4.

### PR DESCRIPTION
**Pull request by paul-amonson**
[Doc] Added build and run documentation.
[Fix] Renamed docker-preview4 to docker-deploy.

Signed-off-by: Paul Amonson <paul.d.amonson@intel.com>

### CODE REVIEW

**Comment by paul-amonson** [comment]
You can see the rendered README.md changes in my fork: https://github.com/paul-amonson/dai-ds/blob/updates_to_readme/README.md

File: README.md [hunk 1]
```diff
@@ -14,28 +14,198 @@ UCS is a flexible software stack that assists development of new designs by OEMs
...
+4. ALL configuration files changed to support the target system. This includes:
+    * SystemManifest.json
+    * MachineConfig.json
+    * NearlineConfig.json
+    * ProviderMonitoringNetworkForeignBus.json
+    * ProviderProvisionerNetworkForeignBus.json
+    * LocationTranslationMap.json - If this doesn't match the machine config
+                                     The provisioning and monitoiring providers
+                                     may not function. Make sure to include ALL Foreign
+                                     names.
```
Comment (nbwhite) [inline README.md]: Several spelling and grammar errors here. Needs to be reworded.

File: README.md [hunk 2]
```diff
@@ -14,28 +14,198 @@ UCS is a flexible software stack that assists development of new designs by OEMs
...
+    systemProp.https.proxyPassword=<proxy_password_if_needed>
+    ```
+    * Python 3's pip requires the proxy variable 'HTTPS_PROXY' is set to
+        the proxy of the environment the host is inside.
+3. Set the following properties in the ~/.gradle/gradle.properties file:
+    ```
+    systemProp.includeDebugSymbols=true
+    ```
+__NOTE:__ Make sure the path to Java 1.8 is correct for the OS distribution
+    being used (OpenSUSE Leap 15.1's path is shown here)!
```
Comment (nbwhite) [inline README.md]: There is no other reference to Java 1.8 in this document. Leftover from previous documents?

File: README.md [hunk 3]
```diff
@@ -14,28 +14,198 @@ UCS is a flexible software stack that assists development of new designs by OEMs
...
+1. sudo systemctl start dai-postgres
+2. sudo systemctl start dai-rabbitmq
+3. sudo systemctl start dai-voltdb
+4. sudo systemctl start dai-manager
+
+__NOTE:__ If you start only the dai-manager service then all other services will
+      start as dependencies if not already running but may experience timing issues and not start up properly.
+
+These services all use the /opt/dai-docker/*.yml files for docker-compose
+launching.  _Do not use docker or docker-compose directly, without out first stopping the services and disabling them_. Attempting to stop a container directly will just cause it to restart as the service is set to restart the container on container exit.
```
Comment (nbwhite) [inline README.md]: "without out", remove out

File: docker-deploy/README.txt [hunk 4]
```diff
@@ -3,27 +3,27 @@ Install and User Guide for DAI on CP4:
...
 
 
 Pre-requisites:
 ----------------
-0. System must have at least 32GB or RAM to run smoothly.
-1. docker installed and configured for the root user.
-2. docker-compose installed.
-3. ALL configuration files changed to support the Preview 4 (or
-   other) system.  This includes:
+1. System must have at least 32GB or RAM to run smoothly.
```
Comment (nbwhite) [inline docker-deploy/README.txt]: Should be "32GB of RAM"

File: README.md (hunk 1 above)
Comment (paul-amonson) [reply README.md]: Resolved with new commit.

File: README.md (hunk 3 above)
Comment (paul-amonson) [reply README.md]: Fixed.

File: docker-deploy/README.txt (hunk 4 above)
Comment (paul-amonson) [reply docker-deploy/README.txt]: Fixed.

**Pull request merged**